packages/lab/src/date-picker/index.ts

```diff
@@ -1,10 +1,14 @@
 export * from "./DatePicker";
 export * from "./DatePickerActions";
 export * from "./DatePickerContext";
+export * from "./DatePickerHelperText";
 export * from "./DatePickerOverlay";
 export * from "./DatePickerOverlayProvider";
 export * from "./DatePickerRangeInput";
+export * from "./DatePickerRangeGridPanel";
 export * from "./DatePickerRangePanel";
 export * from "./DatePickerSingleInput";
-export * from "./DatePickerSinglePanel";
+export * from "./DatePickerSingleGridPanel";
 export * from "./DatePickerTrigger";
+// DatePickerSinglePanel will be deleted when datepicker moves to core
```

Comment:
Only single not range?

Reply:
no Range is the OG panel, that acts as a dual (long range) dates, we keep that and have added a DatePickerRangeGridPanel which is equivalent of DatePickerSingleGridPanel, with a grid of calendars.